docs/components/App.js

```diff
@@ -13,6 +13,25 @@ type Props = {|
 export default function App({ children }: Props): Node {
   const router = useRouter();
 
+  // $FlowIssue[prop-missing]
+  const isLeftClickEvent = (event) => event.button === 0; // ignore everything but left clicks
```

Comment:
Can we add a type for `event` to fix the Flow error?

Reply:
There's a bigger issue here with the event in the provider. 
Leaving FlowIssue as I think I'd have to touch the event in the provider and this can easily get out of scope.
![Screen Shot 2022-03-23 at 4 41 28 PM](https://user-images.githubusercontent.com/10593890/159791825-7f1dd16a-581c-406c-899d-ceb529d1951e.png)

Reply:
There's a bigger issue here with the event in the provider. 
Leaving FlowIssue as I think I'd have to touch the event in the provider and this can easily get out of scope.
![Screen Shot 2022-03-23 at 4 41 28 PM](https://user-images.githubusercontent.com/10593890/159791825-7f1dd16a-581c-406c-899d-ceb529d1951e.png)